core/src/test/java/org/testcontainers/DockerClientFactoryTest.java

```diff
@@ -0,0 +1,46 @@
+package org.testcontainers;
+
+import org.junit.Test;
+import org.testcontainers.dockerclient.LogToStringContainerCallback;
+import org.testcontainers.utility.TestcontainersConfiguration;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created mgorelikov on 05/04/2017
```

Comment:
auto-generated doc comment. I would remove it, doesn't help here.

However, you can include yourself in contributors list :)